druntime/src/__importc_msvc_builtins.d

```diff
@@ -0,0 +1,12888 @@
+/* This file contains D reimplementations of some of the intrinsics recognised
+   by the MSVC compiler, for ImportC.
+   This module is intended for only internal use, hence the leading double underscore.
+
+   Copyright: Copyright D Language Foundation 2024-2024
+   License: $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+   Authors: Harry Gillanders
+   Source: $(DRUNTIMESRC __importc_msvc_builtins.d) */
+
+module __builtins_msvc;
+
+version (CRuntime_Microsoft)
+{
+    version = MSVCIntrinsics;
+}
```

Comment:
Might `version (Windows)` be better for this? Or, is that too broad?

Reply:
`CRuntime_Microsoft` is already used for the `__int<size>` types in `__importc_builtins.di`, so it probably suits here too.